### ISSUE TITLE
Add gamepad support

### ISSUE THREAD

**Issue by ElnuDev** [ElnuDev/hebi#17]

**Comment by grtcdr** [comment]
What's the situation like when it comes to gamepad support?

I know Linux basically provides an interface to pretty much a lot of controllers and gamepads, will you be making direct calls to those interfaces or looking for a more abstracted/convenient wrapper?

**Comment by ElnuDev** [comment]
I have no idea honestly. I think Bevy has gamepad support built-in, but I haven't looked into it yet.

**Comment by grtcdr** [comment]
Interesting, is it cross-platform?

**Comment by ElnuDev** [comment]
It is. I have done builds to both Linux and Windows, and one should in theory be able to build to WebAssembly and mobile as well.

**Comment by grtcdr** [comment]
Wonderful, I'm a controller guy myself.

**Comment by ElnuDev** [comment]
Nice! I'll let you know when we get gamepad support in

**Comment by kroltan** [comment]
> Interesting, is it cross-platform?

Yeah, Bevy uses [`gilrs`](https://crates.io/crates/gilrs) under the hood, From its README:

> ## Supported features
>   | Input | Hotplugging | Force feedback
> -- | -- | -- | --
> Linux | ✓ | ✓ | ✓
> Windows (XInput) | ✓ | ✓ | ✓
> OS X | ✓ | ✓ | ✕
> Wasm | ✓ | ✓ | n/a
> Android | ✕ | ✕ | ✕
